src/node.h

```diff
@@ -637,21 +636,20 @@ class AsyncResource {
         v8::Local<v8::Value>* argv) {
       return node::MakeCallback(isolate_, get_resource(),
                                 symbol, argc, argv,
-                                uid_, trigger_async_id_);
+                                async_context_);
     }
 
     v8::Local<v8::Object> get_resource() {
       return resource_.Get(isolate_);
     }
 
-    async_uid get_uid() const {
-      return uid_;
+    async_id get_uid() const {
```

Comment:
For the sake of keeping things the same, want to rename this to `get_async_id()`?